core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/ngbd-modal-workflow-executions/ngbd-modal-workflow-executions.component.ts

```diff
@@ -31,6 +32,7 @@ export class NgbdModalWorkflowExecutionsComponent implements OnInit {
     "",
     "",
     "Username",
+    "Snapshot",
```

Comment:
lets call it `workflow version sample` or something similar but shorter

Reply:
done